modules/react/breadcrumbs/lib/Breadcrumbs.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import {createContainer} from '@workday/canvas-kit-react/common';
+import {Menu} from '@workday/canvas-kit-react/menu';
+import {useBreadcrumbsModel} from './hooks/useBreadcrumbsModel';
+import {BreadcrumbsList} from './BreadcrumbsList';
+import {BreadcrumbsOverflowButton} from './BreadcrumbsOverflowButton';
+import {BreadcrumbsItem} from './BreadcrumbsItem';
+import {BreadcrumbsLink} from './BreadcrumbsLink';
+import {BreadcrumbsCurrentItem} from './BreadcrumbsCurrentItem';
+import {BreadcrumbsMenu} from './BreadcrumbsMenu';
+
+export interface BreadcrumbsProps extends React.HTMLAttributes<HTMLElement> {
+  /**
+   * The accessibility label for the nav element.
+   *
+   * @default "breadcrumbs"
+   */
+  'aria-label'?: string;
+  /**
+   * The contents of the Breadcrumbs. Can be `Breadcrumbs` children or any valid elements.
+   */
+  children: React.ReactNode;
+}
+
+export const Breadcrumbs = createContainer()({
+  displayName: 'Breadcrumbs',
+  modelHook: useBreadcrumbsModel,
+  subComponents: {
+    List: BreadcrumbsList,
+    Item: BreadcrumbsItem,
+    OverflowButton: BreadcrumbsOverflowButton,
+    CurrentItem: BreadcrumbsCurrentItem,
```

Comment:
```suggestion
    /**
     * ### Breadcrumbs Current Item
     * ---
     * [View Developer Docs](https://canvas.workdaydesign.com/components/navigation/breadcrumbs/#breadcrumbscurrentitem)
     * 
     * The last element in the list of `Breadcrumb.Item`s.
     * By default, this item is truncated with a tooltip at `350px`,
     * But that can be customized with the `maxWidth` prop.
     * 
     * @example
     * ```tsx
     * <Breadcrumbs.CurrentItem maxWidth={200}>
     *   Current Item Text
     * </Breadcrumbs.CurrentItem>
     * ```
     */
    CurrentItem: BreadcrumbsCurrentItem,
```